ComponentKit/Core/Scope/CKComponentScopeHandle.mm

```diff
@@ -14,22 +14,17 @@
 #import "CKComponentControllerInternal.h"
 #import "CKComponentScopeRootInternal.h"
 #import "CKComponentSubclass.h"
+#import "CKComponentInternal.h"
 #import "CKInternalHelpers.h"
 #import "CKMutex.h"
 #import "CKThreadLocalComponentScope.h"
 
 @implementation CKComponentScopeHandle
 {
   id<CKComponentStateListener> __weak _listener;
-  Class _componentClass;
   CKComponentController *_controller;
   CKComponentScopeRootIdentifier _rootIdentifier;
   BOOL _acquired;
-  // Temporarily stored reference to the specific component that acquired this handle. This forms a reference cycle
-  // that is broken in `resolve`. This reference will always be partially initialized, and should not be used outside
-  // the `resolve` call.
```

Comment:
We now just take a weak reference, and we don't remove that reference, so the comment no longer made sense. Removed.